backends-velox/src/test/scala/org/apache/gluten/execution/JsonFunctionsValidateSuite.scala

```diff
@@ -54,9 +54,21 @@ class JsonFunctionsValidateSuite extends FunctionsValidateSuite {
     runQueryAndCompare(
       s"select l_orderkey, json_array_length('[1,2,3,4]') " +
```

Comment:
This case should also read from relation, could you rewrite it as well? 
BTW, ther are same problems with `get_json_object`.

Reply:
ok, I check other tests also.